dask/bag/core.py

```diff
@@ -846,6 +851,29 @@ def from_sequence(seq, partition_size=None, npartitions=None):
     return Bag(d, name, len(d))
 
 
+def from_url(urls):
+    """Create a dask.bag from a url
+
+    >>> a = from_url('http://raw.githubusercontent.com/ContinuumIO/dask/master/README.rst')
+    >>> a.npartitions
+    1
+    >>> 'Dask\n' in a.take(1)
```

Comment:
This looks more like a test than an example.  How about taking more from the README?

```python
>>> a.take(8)
Dask
====

|Build Status| |Coverage| |Doc Status| |Gitter|

Dask provides multi-core execution on larger-than-memory datasets using blocked
algorithms and task scheduling.  It maps high-level NumPy and list operations
```